config/manifests/bases/argocd-operator.clusterserviceversion.yaml

```diff
@@ -660,4 +660,5 @@ spec:
   maturity: alpha
   provider:
     name: Argo CD Community
+  replaces: argocd-operator.v0.0.15
```

Comment:
Did you have to edit the generated CSV to add the correct version (i.e. 0.1.0)?  If so, you can also add the "replaces" field as well and the build/release process have one less file (this file) to touch.

Reply:
No, the correct version was automatically added to the generated CSV.